### PR TITLE
Intro

### PR DESCRIPTION
**Pull request by Johnnycus**
Please create "intro" branch so I can open pull request pointing to that

### CODE REVIEW

File: src/reducer.js [hunk 1]
```diff
@@ -3,13 +3,18 @@ import Immutable from "immutable";
 const init = Immutable.fromJS({
   score: 0,
   gameOver: false,
-  gameStart: true
+  gameStart: true,
+  introSlide: 0
```
Comment (spinlock99) [inline src/reducer.js]: Should this be `currentSlide` instead of `introSlide`?

**Review by spinlock99 — CHANGES_REQUESTED** [review]
This is awesome!

I had a few comments on linting, etc... but overall I'm impressed with the code quality.

File: src/index.ejs [hunk 2]
```diff
@@ -1,25 +1,36 @@
...
+      font-family: "Press Start 2P";
+    }
+
+  </style>
+</head>
+
+<body>
+  <div id="container"></div>
+</body>
+
```
Comment (spinlock99) [inline src/index.ejs]: Same as above. Please keep changes as small as possible.

Can the `font-family` be moved into the component or put on `#container`?

File: src/index.ejs [hunk 3]
```diff
@@ -1,25 +1,36 @@
...
+  <script>addEventListener('error', window.__e = function f(e) { f.q = f.q || []; f.q.push(e) });</script>
+  <script async src="https://www.google-analytics.com/analytics.js"></script>
+  <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Press+Start+2P" rel="stylesheet">
+  <link rel="manifest" href="<%= `${webpackConfig.output.publicPath}manifest.json` %>">
+  <link rel="apple-touch-icon" href="https://process.filestackapi.com/resize=width:144/FmckoTnSQh2QMTsCkPvs">
+  <meta name="apple-mobile-web-app-title" content="Space Force">
+  <meta name="apple-mobile-web-app-capable" content="yes">
+  <meta name="apple-mobile-web-app-status-bar-style" content="black">
+  <meta name="viewport" content="user-scalable=no, width=device-width, initial-scale=1.0, maximum-scale=1.0" />
```
[the remark this answers is not in the document]
Comment (Johnnycus) [reply src/index.ejs]: My bad, haven't checked the settings of my Prettier config. Will fix it!

**Pull request merged**